src/components/manage/Controlpanels/UndoControlpanel.test.jsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import configureStore from 'redux-mock-store';
+import { Provider } from 'react-intl-redux';
+
+import UndoControlpanel from './UndoControlpanel';
+
+const mockStore = configureStore();
+
+jest.mock('react-portal', () => ({
+  Portal: jest.fn(() => <div id="Portal" />),
+}));
+jest.mock('../Form/Form', () => jest.fn(() => <div id="form" />));
+
+describe('UndoControlpanel', () => {
+  it('renders undo controlpanel component', () => {
+    const store = mockStore({
+      transactions: {
+        transactions_recieved: [
+          {
+            description: 'Added default view for root object',
+            id:
+              'QStrR0RRc0M5TjA9\n 2022-06-19T22:15:02 Added default view for root object',
```

Comment:
id should only contain the id, not the full text/description/date

Reply:
Done